src/flamegpu/model/AgentFunctionData.cpp

```diff
@@ -9,33 +9,27 @@ namespace flamegpu {
 
 AgentFunctionData::AgentFunctionData(std::shared_ptr<AgentData> _parent, const std::string &function_name, AgentFunctionWrapper *agent_function, const std::string &in_type, const std::string &out_type)
     : func(agent_function)
-    , rtc_source("")
-    , rtc_func_name("")
     , initial_state(_parent->initial_state)
     , end_state(_parent->initial_state)
     , message_output_optional(false)
     , has_agent_death(false)
     , condition(nullptr)
-    , rtc_condition_source("")
-    , rtc_func_condition_name("")
     , parent(_parent)
-    , description(new AgentFunctionDescription(_parent->description->model.lock(), this))
```

Comment:
This change is obviously temporary and will be updated when Agent Function Description is processed.